common/test/acceptance/tests/lms/test_lms_edxnotes.py

```diff
@@ -677,40 +727,53 @@ def test_tags_view(self):
             time_updated="Jan 01, 2011 at 01:01 UTC"
         )
 
+        self.assert_viewed_event('Tags')
+
     def test_easy_access_from_notes_page(self):
         """
         Scenario: Ensure that the link to the Unit works correctly.
         Given I have a course with 5 notes
         When I open Notes page
         And I click on the first unit link
-        Then I see correct text on the unit page
+        Then I see correct text on the unit page and a unit link event was fired
         When go back to the Notes page
         And I switch to "Location in Course" view
         And I click on the second unit link
-        Then I see correct text on the unit page
+        Then I see correct text on the unit page and a unit link event was fired
+        When go back to the Notes page
+        And I switch to "Tags" view
+        And I click on the first unit link
+        Then I see correct text on the unit page and a unit link event was fired
         When go back to the Notes page
         And I run the search with "Fifth" query
         And I click on the first unit link
-        Then I see correct text on the unit page
+        Then I see correct text on the unit page  and a unit link event was fired
         """
-        def assert_page(note):
+        def assert_page(note, usage_id, view):
+            """ Verify that clicking on the unit link works properly. """
             quote = note.quote
             note.go_to_unit()
             self.courseware_page.wait_for_page()
             self.assertIn(quote, self.courseware_page.xblock_component_html_content())
+            self.assert_unit_link_event(usage_id, view)
+            self.reset_event_tracking()
```

Comment:
nit: I find it more clear to use the context manager than to call reset_event_tracking, but realize that's a personal preference thing (see the bottom of this section: http://edx.readthedocs.org/projects/edx-developer-guide/en/latest/analytics.html#bok-choy-testing)

no need to actually change anything here